packages/compile-common/src/index.ts

```diff
@@ -1,2 +1,62 @@
 import { Profiler } from "./profiler";
 export { Profiler };
+
+type Compilation = {
```

Comment:
gotta export all these!

Reply:
Are we going to use these anywhere?

Reply:
almost certainly :)

Reply:
I mean immediately. I was going to wait until it was used to export it but if you think we should do it now then I'll change it.